actions/createMultisigWallet.ts

```diff
@@ -1,35 +1,25 @@
-import { Connection, PublicKey } from '@solana/web3.js'
-
 import {
   sendTransactionsV3,
   SequenceType,
   txBatchesToInstructionSetWithSigners,
-  WalletSigner,
 } from 'utils/sendTransactions'
 import { chunks } from '@utils/helpers'
-import { prepareRealmCreation } from '@tools/governance/prepareRealmCreation'
+
+import {
+  prepareRealmCreation,
+  RealmCreation,
+  Web3Context,
+} from '@tools/governance/prepareRealmCreation'
 import { trySentryLog } from '@utils/logs'
 
 /// Creates multisig realm with community mint with 0 supply
 /// and council mint used as multisig token
-interface MultisigWallet {
-  connection: Connection
-  wallet: WalletSigner
-  programIdAddress: string
-
-  realmName: string
-  councilYesVotePercentage: number
-  councilWalletPks: PublicKey[]
-}
+type MultisigWallet = RealmCreation & Web3Context
```

Comment:
I unified the input types for the wizards, and migrated all form -> parameter transformation to one place instead of having it spread across multiple functions randomly.